gspread/utils.py

```diff
@@ -8,7 +8,7 @@
 
 import re
 from collections import defaultdict
-from collections.abc import Sequence
+from collections.abc import Mapping, Sequence
```

Comment:
you should import `Mapping` from `typing` instead. This is why the Python 3.8 test fails.